### DISCUSSION TITLE
Favorite folders

### DISCUSSION THREAD

**Discussion by fax13** [music-assistant/support#906] [Feature requests and ideas]
Hi, what about favorite folder and/or artists/albums favorites  possibility.  Would be a time saver

**Comment by DutchJaFO** [comment]
As far as I know the 'heart' button already acts like a 'favourite' selection ... but it is kind of useless if you like 'everything' you own already 

Possible alternative : create .m3u files for your favourites. 
Your playlist with your favourites will appear in the 'playlist' category with "Filesystem" in light gray text.
![image](https://user-images.githubusercontent.com/39138434/189519505-8fd99949-3614-4a3d-851a-466ba41b9efc.png)

You may need to sync your library for it to appear.

**Reply by fax13** [reply]
my appologies for not to be more detailed reagrding my question.
i wanted to reffer to nas files and folders whereby favorite folders can be shortcutted. i presume that when music is stored on nas systems, most likely it contains different folders. could be very helpfull when playlists are build so that it is possible to jump faster for location to location.

**Reply by OzGav** [reply]
The docs describe how the music folder structure should be set up so I assume that within that you are wanting to favourite some folder (eg. An artist you like). I am not sure what Marcel will think about this as the purpose of MA is to abstract away the detail of folders and multiple providers to provide one single interface with all available music?

**Comment by OzGav** [comment]
Old v1 question and implemented in v2
